dali/pipeline/operator/builtin/external_source_test.cc

```diff
@@ -147,7 +148,9 @@ class ExternalSourceTest : public::testing::WithParamInterface<int>,
       vt_gpu_[j].Copy(tensor);
     }
     CUDA_CALL(cudaStreamSynchronize(0));
-    src_op->SetDataSource(vt_gpu_);
+    ExtSrcSettingMode settings = {};
+    settings.sync = true;
+    src_op->SetDataSource(vt_gpu_, {}, settings);
```

Comment:
Does it mean that `SetDataSource` behaves differently now?

Reply:
Good question. Need to recheck.

Reply:
OK - if I set the access order as stream 0 for the TL, it works. Note that GPU TLs produced by DALI will have a stream set. I don't know how we create TLs for the GPU external source, but nothing seemed to be broken...

Reply:
We can use `__cuda_array_interface__` or dlpack for that https://github.com/NVIDIA/DALI/blob/main/dali/python/backend_impl.cc#L912
Btw - I see that v3 adds stream. Maybe we should add it support as well https://numba.readthedocs.io/en/stable/cuda/cuda_array_interface.html?

Reply:
Not this PR, maybe? It seems to work the way it is.

Reply:
Ok